app/containers/Admin/ReportingYearTable.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import {Table} from 'react-bootstrap';
+import {graphql, createFragmentContainer} from 'react-relay';
+import {ReportingYearTable_query} from '__generated__/ReportingYearTable_query.graphql';
+
+interface Props {
+  query: ReportingYearTable_query;
+}
+
+function formatDate(date) {
+  const d = new Date(date);
+  return d.toLocaleString();
```

Comment:
We are using the `moment-timezone` library for rendering of dates.
For example;
https://github.com/bcgov/cas-ciip-portal/blob/4ec58fc2e97ee12a81d4d4544ce38713bfb0e03c/app/pages/index.tsx#L39-L45